src/messages/Message.hpp

```diff
@@ -33,6 +33,7 @@ enum class MessageFlag : uint32_t {
     Whisper = (1 << 16),
     HighlightedWhisper = (1 << 17),
     Debug = (1 << 18),
+    Hidden = (1 << 19),
```

Comment:
This flag should be called something with "Similar" or something instead, since it doesn't really mean hidden, it just means that this message was marked as too similar to previous messages according to your settings.

Reply:
renamed